examples/4.safe-route.js

```diff
@@ -0,0 +1,60 @@
+import axios from "axios";
+import utils from "../utils.js";
+import Safe from "@safe-global/protocol-kit";
+import { EthersAdapter } from "@safe-global/protocol-kit";
+import * as ethers from "ethers";
+import { OperationType } from "@safe-global/safe-core-sdk-types";
+
+// in this example, we will execute a route via a safe wallet
+
+const approveDaiToAdai = async () => {
```

Comment:
This name doesnt seem correct, maybe a copy-paste?

Reply:
oh yeah, do you care to update it for me?